.travis.yml

```diff
@@ -26,6 +24,6 @@ before_script:
 
 script:
   - npm run build --prefix client/
-  - mvn clean test
+  - mvn clean test jacoco:report coveralls:report
```

Comment:
i think, you have add this command to command "build", only in this case it will be executing by Trevis CI

Reply:
it seems that everything is ok here. You should add to build only those commands, which are executed during front-end build